### PR TITLE
Give "Reason" a reference tooltip

### PR DESCRIPTION
**Pull request by arkadyan**
Add a question mark icon.

Screen Shot 2020-03-10 at 14 51 49

Screen Shot 2020-03-11 at 15 02 23

### CODE REVIEW

**Comment by arkadyan** [comment]
This is currently on dev for testing (the tooltip will not show up running in your local development server).

**Comment by ingridpierre** [comment]
@arkadyan can we make "Reason" display as "REASON" to match the other little labels below?

**Comment by arkadyan** [comment]
@ingridpierre Good catch, I fixed the uppercase thing.

**Comment by ingridpierre** [comment]
One more lil tweak, current waivers should use green in the label. like so:

Current:
image
Desired:
image

**Comment by arkadyan** [comment]
@ingridpierre @skyqrose Updated to use a local tooltip instead of Appcues.

**Comment by ingridpierre** [comment]
"Operator Error" should be "Operator error" for consistency.

File: assets/css/properties_panel/_block_waiver_banner.scss [hunk 1]
```diff
@@ -101,3 +101,45 @@
...
+
+  .m-block-waiver-banner--future &,
+  .m-block-waiver-banner--past & {
+    fill: $color-font-grey;
+  }
+}
+
+.m-block-waiver-banner__reason-tooltip.__react_component_tooltip.show {
+  opacity: 1;
+  text-align: left;
```
Comment (skyqrose) [inline assets/css/properties_panel/_block_waiver_banner.scss]: With the ladder styling moved back to be specifically about the ladder, this property isn't needed anymore.

File: assets/src/skate.d.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ declare global {
     Appcues?: {
       identify: (userId: string) => void
       page: () => void
+      show: (id: string) => void
```
Comment (skyqrose) [inline assets/src/skate.d.ts]: I don't see this function called anywhere. What happens to the tests if it's not mocked?

**Comment by arkadyan** [comment]
@ingridpierre thanks, made error lowercase.

File: assets/css/properties_panel/_block_waiver_banner.scss (hunk 1 above)
Comment (arkadyan) [reply assets/css/properties_panel/_block_waiver_banner.scss]: Yep, removed.

File: assets/src/skate.d.ts (hunk 2 above)
Comment (arkadyan) [reply assets/src/skate.d.ts]: Yep, that is no longer needed.

**Comment by ingridpierre** [comment]
@arkadyan lgtm 👍

**Pull request merged**